### DISCUSSION TITLE
Post Cache always failed

### DISCUSSION THREAD

**Discussion by hantsy** [shivammathur/setup-php#763] [Q&A / Help] [answered]
On Github actions, I always encountered this issue.

https://github.com/hantsy/symfony-rest-sample/actions/runs/6071754673/job/16470411755

```bash
Error: The template is not valid. .github/workflows/build.yml (Line: 68, Col: 16): 
hashFiles('**/composer.lock') failed. Fail to hash files under directory 
'/home/runner/work/symfony-rest-sample/symfony-rest-sample'
```

My Github actions workflow is modified from the Symfony example:

https://github.com/hantsy/symfony-rest-sample/blob/master/.github/workflows/build.yml

**Comment by shivammathur** [comment, marked as answer]
@hantsy 

Try with '/composer.lock'.

```diff
- key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+ key: ${{ runner.os }}-composer-${{ hashFiles('/composer.lock') }}
```

**Reply by hantsy** [reply]
Worked, thanks.
